set-based-validation/src/main/java/io/axoniq/dev/samples/handler/AccountEventHandler.java

```diff
@@ -0,0 +1,30 @@
+package io.axoniq.dev.samples.handler;
```

Comment:
You could for clarity move this class, together with the Aggregate, repository and entity, into a grouped `command` package. That would group all Command Side components into a distinct package.